### BIG differences, this is diff of wrong files.
### CLUS "Function VECTOR"
### and CLHS topic selection http://clhs.lisp.se/Body/a_vector.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:vector.txt	2017-04-27 22:04:53.000000000 +0100
+++ saved-pages-clus/cl:functions:vector.txt	2017-04-27 22:04:54.000000000 +0100
@@ -1,2 +1,1 @@
-VECTOR
-Please select which reference to VECTOR you intended
+Function VECTOR

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Syntax
+ vector &rest objects → vector
+Arguments and Values
+ object - an object.
+ vector - a vector of type (vector t *).
+Description
+Creates a fresh simple general vector whose size corresponds to the number of objects.
+The vector is initialized to contain the objects.
+Examples
+(defparameter *v* (vector 1 2 'sirens))
+→
+*V*
+(arrayp *v*)
+→
+true
+(vectorp *v*)
+→
+true
+(simple-vector-p *v*)
+→
+true
+(length *v*)
+→
+3
+Side Effects
+None.
+Affected By
+None.
+Exceptional Situations
+None.
+See Also
+Function MAKE-ARRAY
+Notes
+vector is analogous to list.
+(vector a1 a2 ... an)
+  ≡ (make-array ([CLFunctionslist]] n)
+                element-type t
+                initial-contents (list a1 a2 ... an)))
\ No newline at end of file
